wcvendors/wcvendors
# Fixed: Order export fails if the order has been deleted #813

### All Submissions:

### Changes proposed in this Pull Request:

Adds a check to the shipped column to ensure the order exists

Closes #813

### How to test the changes in this Pull Request:

1. Go to commissions after deleting an order
2. Try to export
3. See export works
